lms/resources/_js_config.py

```diff
@@ -42,6 +42,9 @@ def config(self):
             # The auth token that the JavaScript code will use to authenticate
             # itself to our own backend's APIs.
             "authToken": self._auth_token(),
+            # The URL that the JavaScript code will open if it needs the user to
+            # authorize us to request a new access token.
+            "authUrl": self._request.route_url("canvas_api.authorize"),
```

Comment:
Worth adding a note to mention this is Canvas specific?

Reply:
Added a comment